src/components/tabs/tabs.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react"
+import { storiesOf } from "@storybook/react"
+
+import { action } from "@storybook/addon-actions"
+import { text, boolean, select, number } from "@storybook/addon-knobs"
+import { readmeCleanup } from "../../../utils/readme"
+import { Tabs, Tab } from "."
+// @ts-ignore
+import readme from "./README.md"
+
+const tabsStory = storiesOf("COMPONENTS|Tabs", module)
+
+const subData = {
+  readme: {
+    sidebar: readmeCleanup(readme),
+  },
+  jest: ["tabs.test.tsx"],
+}
+
+const makeOnChange = {
+  no: undefined,
+  yes: action("Tab changed"),
+}
+
+const TabContentExample = props => <i>all in italics: {props.children}</i>
+
+const makeTabContentSelect = {
+  TabContentExample,
+  b: "b",
+}
+
+tabsStory.add(
+  "Tabs",
```

Comment:
Let's discuss a case, where we have a limited space for a component using tabs - like, 720px. And the quantity/width of tab "headings" takes more width.  Will tabs "scale", with ellipsis-cutting text, or offer a horizontal scroll?